storages/csv-storage/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "gluesql_csv_storage"
```

Comment:
It would be better to use -.

```suggestion
name = "gluesql-csv-storage"
```

Reply:
```toml
[package]
name = "gluesql_memory_storage"
```
Hmm... but other storage subcrates in this project use snake case.

Reply:
memory storage has been deployed before so it is :cry:

Reply:
ref: <https://github.com/gluesql/gluesql/blob/main/storages/shared-memory-storage/Cargo.toml#L2>

Reply:
Oh, you don't need to care about underscore cases. You can use `-` dash.
I published `gluesql_memory_storage` and `gluesql_sled_storage` before we change our project structure into workspace-based.
And.. `crates.io` does not distinguish `_` and `-` so I wasn't able to rename `gluesql_memory_storage` to `gluesql-memory-storage`.


![image](https://user-images.githubusercontent.com/2025065/178138934-684b2e08-c800-420c-9455-19a2606b8fe3.png)